components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -217,7 +217,7 @@ export default class SearchBar extends mixins(
           events: [{ meta; timestamp; nft }]
         }>({
           query: lastNftListByEvent,
-          client: 'legacysquid',
+          client: this.urlPrefix === 'rmrk' ? 'subsquid' : this.urlPrefix,
```

Comment:
```suggestion
          client: this.client,
```

Reply:
why?

Reply:
> why?

PrefixMixin refactoring

Reply:
> PrefixMixin refactoring

kewl

Reply:
> `this.client` won't make sense for the code that is prefix sensitive :) so may have both?


I'm not sure I understood what you said but in `PrefixMixin` we have `client()` = `this.urlPrefix === 'rmrk' ? 'subsquid' : this.urlPrefix` so it will be the same as before

Reply:
Perfect!